examples/nlp/language_modeling/conf/megatron_t5_config_finetune_glue_eval.yaml

```diff
@@ -0,0 +1,34 @@
+name: megatron_t5_glue_eval
+
+trainer:
+  gpus: 8
+  num_nodes: 1
+  accelerator: ddp
+  precision: 16
+  logger: False # logger provided by exp_manager
+  checkpoint_callback: False
+  replace_sampler_ddp: False
+  log_every_n_steps: 10
+
+
+exp_manager:
+  explicit_log_dir: null
+  exp_dir: null
+  name: megatron_t5_glue_eval
+  create_checkpoint_callback: False
+
+model:
+  restore_from_finetuned_path: ???.nemo # Path to a trained T5 .nemo file
```

Comment:
I think it's better to leave this is as just ??? instead of ???.nemo since hydra will treat it as a mandatory field?

Reply:
yeah, that makes sense